ftests/core/src/main/java/org/commonjava/indy/ftest/core/content/SetContentTypeFromUpstreamIfExistsTest.java

```diff
@@ -0,0 +1,69 @@
+package org.commonjava.indy.ftest.core.content;
+
+import org.apache.http.HttpResponse;
+import org.commonjava.indy.client.core.IndyClientModule;
+import org.commonjava.indy.client.core.helper.HttpResources;
+import org.commonjava.indy.client.core.module.IndyRawHttpModule;
+import org.commonjava.indy.ftest.core.AbstractContentManagementTest;
+import org.commonjava.indy.model.core.RemoteRepository;
+import org.junit.Test;
+
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static org.commonjava.indy.model.core.StoreType.remote;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
```

Comment:
We need to add some "README" of ftests to show what this ftest is for. We have standard format for this, like "GIVEN, WHEN, THEN". You can refer some of the existing ftests to see how it looks like.

Reply:
Cool, i will go through the standard and update it later.